### PR TITLE
metastatus now prints an error when marathon is configured & unreachable

### PR DESCRIPTION
**Pull request by mjksmith**

### CODE REVIEW

File: paasta_tools/paasta_metastatus.py [hunk 1]
```diff
@@ -349,8 +350,12 @@ def main():
         chronos_results = [('chronos is not configured to run here', True)]
 
     if marathon_config:
-        marathon_client = get_marathon_client(marathon_config)
-        marathon_results = get_marathon_status(marathon_client)
+        try:
+            marathon_client = get_marathon_client(marathon_config)
+            marathon_results = get_marathon_status(marathon_client)
+        except MarathonError:
```
Comment (solarkennedy) [inline paasta_tools/paasta_metastatus.py]: Can you make sure the error text is still printed?

**Comment by solarkennedy** [comment]
fix +ship

File: paasta_tools/paasta_metastatus.py [hunk 2]
```diff
@@ -349,8 +350,12 @@ def main():
         chronos_results = [('chronos is not configured to run here', True)]
 
     if marathon_config:
-        marathon_client = get_marathon_client(marathon_config)
-        marathon_results = get_marathon_status(marathon_client)
+        try:
+            marathon_client = get_marathon_client(marathon_config)
+            marathon_results = get_marathon_status(marathon_client)
```
Comment (Rob-Johnson) [inline paasta_tools/paasta_metastatus.py]: only wrap ``marathon_results = get_marathon_status(marathon_client)`` in the try/except here - creating the client doesn't throw an exception.

**Comment by Rob-Johnson** [comment]
whilst you're here, could you do the same for chronos?

**Pull request merged**